myapp/config/locales/ja.yml

```
@@ -0,0 +1,11 @@
+ja:
+  messages:
+    create: '%{model_name}の登録に成功しました。'
+    update: '%{model_name}の更新に成功しました。'
+    delete: '%{model_name}の削除に成功しました。'
+
+  error:
+    messages:
+      create: '%{model_name}の登録に失敗しました。'
+      update: '%{model_name}の更新に失敗しました。'
+      record_not_found: '該当する%{model_name}がありませんでした。'
```

[修正任意]
```suggestion
      record_not_found: '該当するリソースがありませんでした。'
```
ここは、model_name 使わなくて良いかなと思いました。